acme/acme/mixins.py

```diff
@@ -30,6 +30,8 @@ class ResourceMixin(VersionedLEACMEMixin):
     This mixin generates a RFC8555 compliant JWS payload
     by removing the `resource` field if needed (eg. ACME v2 protocol).
     """
+    resource_type: str = NotImplemented
```

Comment:
I think this shouldn't be needed anymore.

Reply:
Indeed, fixed.